thunder/torch/__init__.py

```diff
@@ -2845,6 +2845,39 @@ def _normalize(a: TensorProxy, /, norm_dims, eps: Number) -> tuple[TensorLike, T
     return out, mean, rstd
 
 
+@torchsymbol(torch.nn.functional.normalize, is_method=True)
+def normalize(
+    a: TensorProxy, /, p: float = 2.0, dim: int | Sequence[int] = 1, eps: float = 1e-12, out: None | TensorProxy = None
+) -> TensorProxy:
+    utils.check(
+        dtypes.is_float_dtype(a.dtype) or dtypes.is_complex_dtype(a.dtype),
+        lambda: f"normalize: Expected a floating point or complext tensor as input. Got {a.dtype}",
+        TypeError,
+    )
+    utils.check(out is None, lambda: "normalize: out is not None which is currently unsupported", NotImplementedError)
+    computation_dtype, result_dtype = _reduction_dtypes(a, REDUCTION_OUTPUT_TYPE_KIND.COMPLEX_TO_FLOAT, a.dtype)
+    if p == 0.0:
+        return a / sum(ne(a, 0.0), dim=dim, keepdim=True)
```

Comment:
Would this work?
```suggestion
        denom = sum(a != 0.0, dim=dim, keepdim=True)
```

Reply:
Yup! I don't see any problem with this. It's also more consistent with the other part so let me accept this! Thank you again!